config/templates/blueprint-manifest.yml.ejs

```diff
@@ -34,21 +30,36 @@ if (!spec.previous_manifest) {
     }
   };
 } else {
-  /* update scenario: preserve previous service instance properties */
-  properties.blueprint = spec.previous_manifest.properties.blueprint;
+  if(!spec.previous_manifest.instance_groups){
+    properties.blueprint = spec.previous_manifest.properties.blueprint;
+  }else{
+    properties.blueprint = spec.previous_manifest.instance_groups[0].jobs[0].properties;
```

Comment:
In your testing didnt you find this was to be named as `spec.previous_manifest.instance_groups[0].jobs[0].properties.blueprint` - ? Just checking

Reply:
that was a mistake in the test implementation. I fixed it.
In bosh v2 manifest schema, we don't use the level 'blueprint' and place properties under it, the way we were doing in the bosh v1 schema. I had missed that part in my test mocks.

Reply:
ok.